src/mongocrypt.h.in

```diff
@@ -408,6 +408,22 @@ bool
 mongocrypt_setopt_schema_map (mongocrypt_t *crypt,
                               mongocrypt_binary_t *schema_map);
 
+/**
+ * Set a local EncryptedFieldConfigMap for encryption.
+ *
+ * @param[in] crypt The @ref mongocrypt_t object.
+ * @param[in] schema_map A BSON document representing the EncryptedFieldConfigMap supplied by
+ * the user. The keys are collection namespaces and values are EncryptedFieldConfigMap documents. The
+ * viewed data copied. It is valid to destroy @p efc_map with @ref
```

Comment:
"The viewed data copied" isn't really a sentence. Should this be "The config bson is copied" or something like that?